frontend/.prettierrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "trailingComma" : "none"
```

Comment:
The single quote is working correctly @Cerberuuus?

Reply:
Yes, as it is in the _tslint.json_.
```json
 "quotemark": [
      true,
      "single"
    ],
```

However, the above configuration may not be needed, if you think that trailing commas are not confusing and shouldn't be ignored.

Reply:
Yes, I thought to the default formatter settings. But if you do not found a quote problem, then it is ok.